api/dao/base.py

```diff
@@ -80,12 +80,17 @@ def get_container(self, _id, projection=None, get_children=False):
             cont[CHILD_MAP[self.cont_name]] = children
         return cont
 
-    def get_children(self, _id, projection=None):
+    def get_children(self, _id, projection=None, uid=None):
         try:
             child_name = CHILD_MAP[self.cont_name]
         except KeyError:
             raise APINotFoundException('Children cannot be listed from the {0} level'.format(self.cont_name))
-        query = {self.cont_name[:-1]: bson.ObjectId(_id)}
+        if self.cont_name == 'groups':
```

Comment:
We should use `self.use_object_id` here instead of keying directly on groups, right now there isn't a difference because the other container that don't use an object id don't ever call get_children